Client/Help/src/app/components/content-messages-gallery/content-messages-gallery.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import {faUserCircle } from '@fortawesome/free-solid-svg-icons';
+import {UserAccountService} from '../../services/user-account.service';
+
+@Component({
+  selector: 'app-content-messages-gallery',
+  templateUrl: './content-messages-gallery.component.html',
+  styleUrls: ['./content-messages-gallery.component.css']
+})
+export class ContentMessagesGalleryComponent implements OnInit {
+  profilePic = faUserCircle;
+  arrayOfProfiles = this.userAccountService.getConnections();
```

Comment:
It's probably worth setting up fixture data in your service, so that some connections have messages. Then, you could call a different method here that only returns connections that have been messaged. (At least, that's how I interpret this screen to be used.)